js/app.js

```diff
@@ -252,7 +252,9 @@ const main = () => {
           $scope.window_type = 'track';
           return;
         }
-        const listId = (new URL(url, window.location)).searchParams.get('list_id');
+        const listId = new URL(url, window.location).searchParams.get(
```

Comment:
Does this format change due to "eslint-config-prettier": "^8.1.0", update? 
It seems that rule has changed but no eslint or prettier rule config change found.

Reply:
The line was introduced in the latest commit and doesn't contains the formatting of that code. 
Prettier might think that line is too long.

Reply:
got it. If there's some method to test whetther commit code has used prettier will very helpful.